docs/sources/setup/install/helm/deployment-guides/aws.md

```diff
@@ -0,0 +1,763 @@
+    loki-compactor-0                        1/1     Running   0          10m
+    loki-distributor-78ccdcc9b4-9wlhl       1/1     Running   0          10m
+    loki-distributor-78ccdcc9b4-km6j2       1/1     Running   0          10m
+    loki-distributor-78ccdcc9b4-ptwrb       1/1     Running   0          10m
+    loki-gateway-5f97f78755-hm6mx           1/1     Running   0          10m
+    loki-index-gateway-0                    1/1     Running   0          10m
+    loki-index-gateway-1                    1/1     Running   0          10m
+    loki-ingester-zone-a-0                  1/1     Running   0          10m
+    loki-ingester-zone-b-0                  1/1     Running   0          10m
+    loki-ingester-zone-c-0                  1/1     Running   0          10m
+    loki-querier-89d4ff448-4vr9b            1/1     Running   0          10m
+    loki-querier-89d4ff448-7nvrf            1/1     Running   0          10m
+    loki-querier-89d4ff448-q89kh            1/1     Running   0          10m
+    loki-query-frontend-678899db5-n5wc4     1/1     Running   0          10m
+    loki-query-frontend-678899db5-tf69b     1/1     Running   0          10m
+    loki-query-scheduler-7d666bf759-9xqb5   1/1     Running   0          10m
+    loki-query-scheduler-7d666bf759-kpb5q   1/1     Running   0          10m
+    loki-results-cache-0                    2/2     Running   0          10m
+    loki-ruler-0                            1/1     Running   0          10m
+    ```
+  
+### Find the Loki Gateway Service
+
+The Loki Gateway service is a LoadBalancer service that exposes the Loki gateway to the internet. This is where you will write logs to and query logs from. By default NGINX is used as the gateway.
+
+{{< admonition type="caution" >}}
+The Loki Gateway service is exposed to the internet. We provide basic authentication using a username and password in this tutorial. Refer to the [Authentication]({{< relref "../../../../operations/authentication" >}}) documentation for more information.
+{{< /admonition >}}
+
+To find the Loki Gateway service, run the following command:
+
+```bash
+kubectl get svc -n loki
+```
+You should see the Loki Gateway service with an external IP address. This is the address you will use to write to and query Loki.
+
+```console
+  NAME                             TYPE           CLUSTER-IP       EXTERNAL-IP                                                               PORT(S)              AGE
+loki-gateway                     LoadBalancer   10.100.201.74    12345678975675456-1433434453245433545656563.eu-west-2.elb.amazonaws.com   80:30707/TCP         46m
+```
+
+Congratulations! You have successfully deployed Loki on AWS using the Helm chart. Before we finish, let's test the deployment.
+
+## Testing Your Loki Deployment
+
+k6 is one of the fastest way to test your Loki deployment. This will allow you to both write and query logs to Loki. To get started with k6, follow the steps below:
```

Comment:
```suggestion
k6 is one of the fastest ways to test your Loki deployment. This will allow you to both write and query logs to Loki. To get started with k6, follow the steps below:
```